Core/Core/RichContentEditor/RichContentEditor.css

```diff
@@ -1,5 +1,10 @@
+html {
+    background: var(--color-backgroundLightest);
+    color: var(--color-textDarkest);
+    font-family: system-ui;
+    font-weight: 500;
+}
 html, body {
-    height: 100%;
```

Comment:
This made the content fill any remaining space in the webView, but it also broke re-shrinking in autoresizeHeight. As it turns out, tapping in the webView but after the html element still focuses the content, so I don't think this is really needed anyway.